web-ui/src/pages/OnboardProgressPage.jsx

```diff
@@ -147,7 +178,8 @@ export default function OnboardProgressPage() {
           open={open}
           onClose={handleClose}
           aria-labelledby="title"
-          aria-describedby="description">
+          aria-describedby="description"
+        >
           <Box sx={modalBoxStyle}>
             <Typography align="center" id="title" variant="h3" component="h2">
```

Comment:
Typography tag needs to be closed somewhere? </Typography>

Reply:
It's not what I wrote, its from a commit from before

Reply:
so Im guessing its okay to leave it open?

Reply:
Let's close any missing tags.

Reply:
I checked again, they were all already closed. Its not showing up for you because the parts where I didnt code were cut out from the view on Github